lib/src/request.dart

```diff
@@ -130,6 +133,28 @@ class Request extends Message {
       {Map<String, String> headers, Map<String, Object> context})
       : this('DELETE', url, headers: headers, context: context);
 
+  factory Request.multipart(url,
+      {Map<String, String> headers,
+        Map<String, Object> context,
+        Map<String, String> fields,
+        Iterable<MultipartFile> files}) {
+    fields ??= <String, String>{};
+    files ??= <MultipartFile>[];
+    headers ??= <String, String>{};
```

Comment:
Doesn't context need a default value?

Reply:
It will end up getting set when a `HttpUnmodifiableMap` is created within the `Message` constructor so no it doesn't need one.